test/lib/tasks/minify.js

```diff
@@ -1,22 +1,24 @@
 import test from "ava";
 import sinonGlobal from "sinon";
 import esmock from "esmock";
+import {createTaskUtil} from "../../utils/taskUtilHelper.js";
 
 test.beforeEach(async (t) => {
 	const sinon = t.context.sinon = sinonGlobal.createSandbox();
 	t.context.workspace = {
 		byGlob: sinon.stub().resolves(["resource A", "resource B"]),
 		write: sinon.stub().resolves()
 	};
-	t.context.taskUtil = {
+	t.context.taskUtil = createTaskUtil(t, {
```

Comment:
I'm afraid these changes can be reverted, right? `createFilterReader` is no longer used